js/repl/Repl.js

```diff
@@ -567,13 +577,12 @@ class Repl extends React.Component<Props, State> {
 
     const presetsArray = this._presetsToArray();
 
-    const builtIns = envConfig.isBuiltInsEnabled && envConfig.builtIns;
```

Comment:
The are moved to `_onOptionChange` so the `envPresetDefaults.builtIns.default` will be also sync to UI.